index.html

```diff
@@ -222,6 +222,107 @@ <h2 class="portfolio__title">Portfolio</h2>
+                        <div class="testimonial__image">
+                            <img class="testimonial__img" src="./images/Testimonial/ProfilePicture.svg" alt="Testimonial Image">
+                        </div>
+                        <div class="testimonial__text">
+                            <div class="testimonial__comment__box">
+                                <img class="testimonial__quatation" src="./images/Testimonial/icons/top-quotation.svg" alt="Quatation Icon">
+                                <p class="testimonial__comment">But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was born and I will give you a complete</p>
+                                <img class="testimonial__quatation right" src="./images/Testimonial/icons/bot-quotation.svg" alt="Quatation Icon">
+                            </div>
+                            <div class="testimonial__comment__author">
+                                <p class="testimonial__name">Jared Warner</p>
+                                <p class="testimonial__desc">CEO Of Zeo</p>
+                            </div>
+                            <div class="testimonial__buttons">
+                                <div class="testimonial__boxes__pagination"></div>
+                                <div class="testimonial__arrows arrows">
+                                    <img class="swiper-button-prev testimonial-button" src="./images/Swiper/arrow-left.svg" alt="Previous"/>
+                                    <img class="swiper-button-next testimonial-button" src="./images/Swiper/arrow-right.svg" alt="Next"/>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="swiper-slide testimonial__content">
+                        <div class="testimonial__image">
+                            <img class="testimonial__img" src="./images/Testimonial/ProfilePicture.svg" alt="Testimonial Image">
+                        </div>
+                        <div class="testimonial__text">
+                            <div class="testimonial__comment__box">
+                                <img class="testimonial__quatation" src="./images/Testimonial/icons/top-quotation.svg" alt="Quatation Icon">
+                                <p class="testimonial__comment">But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was born and I will give you a complete</p>
+                                <img class="testimonial__quatation right" src="./images/Testimonial/icons/bot-quotation.svg" alt="Quatation Icon">
+                            </div>
+                            <div class="testimonial__comment__author">
+                                <p class="testimonial__name">Jared Warner</p>
+                                <p class="testimonial__desc">CEO Of Zeo</p>
+                            </div>
```

Comment:
`.testimonial__name` and `.testimonial__desc` are not really good class names. Is that the name and desc of the testimonial or the author of the comment?